src/Routes.tsx

```diff
@@ -1,12 +1,14 @@
 import { Routes, Route, Navigate } from 'react-router-dom';
 import DiscordLogin from './components/buttons/DiscordLogin';
+import Detail from './pages/Detail';
 import Home from './pages/Home';
 
 const _Routes = () => {
   return (
     <Routes>
       <Route path="/" element={<Home />}>
         <Route index element={<DiscordLogin />} />
+        <Route path="/detail" element={<Detail />} />
```

Comment:
[IMO]
<Route path="/detail" element={<Detail />} />

```suggestion
        <Route path="detail" element={<Detail />} />
```

위처럼 해도 작동은 하지만 아래의 형태가 좀 더 이해하기 쉬운 것 같아요. 마치 `//` 형태로 중복되어있는 것 처럼 생각되네여.